app/controllers/chapters_controller.rb

```diff
@@ -216,18 +216,25 @@ def destroy
     if @chapter.is_only_chapter?
       flash[:error] = ts("You can't delete the only chapter in your story. If you want to delete the story, choose 'Delete work'.")
       redirect_to(edit_work_path(@work))
+      return
+    end
+
+    if @chapter.only_posted_chapter?
+      flash[:error] = ts("You can't delete the only published chapter in your story. Please post another chapter first. If you want to delete the story, choose 'Delete work'.")
```

Comment:
Could you update this to match the text provided in the issue?

> Sorry, you can't delete the only posted chapter of a work. Please post another chapter first. You can also choose to delete the work.

The existing error message uses outdated terminology (we try to avoid assuming works are stories) and style (we prefer double quotes), so if you could update that when you move it to the locale file, we'd appreciate it:

>You can't delete the only chapter in your work. If you want to delete the work, choose "Delete Work".